### ISSUE TITLE
[HW] Jasbir Singh

### ISSUE THREAD

**Issue by jasbir84** [datastaxdevs/k8ssandra-workshop#4]
**Name:** Jasbir Singh
**Linkedin Profile:** https://linkedin.com/in/jasbir84

Attach the homework screenshots below:
-----------------------------------------
Homework using local kind cluster on mac:

<img width="1680" alt="Screenshot 2021-03-04 at 3 21 53 PM" src="https://user-images.githubusercontent.com/12182961/109945946-c21a3680-7cfd-11eb-81c9-3b65fd337e7a.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 22 00 PM" src="https://user-images.githubusercontent.com/12182961/109945952-c34b6380-7cfd-11eb-99b4-62d402dbde53.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 22 24 PM" src="https://user-images.githubusercontent.com/12182961/109945958-c47c9080-7cfd-11eb-860e-9babf861c5d5.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 22 49 PM" src="https://user-images.githubusercontent.com/12182961/109945965-c5adbd80-7cfd-11eb-9ae2-114b00cf5e78.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 23 09 PM" src="https://user-images.githubusercontent.com/12182961/109945972-c6deea80-7cfd-11eb-8e73-94fe4839a10a.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 23 59 PM" src="https://user-images.githubusercontent.com/12182961/109945979-c8a8ae00-7cfd-11eb-99a4-e6e04fa3243b.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 24 16 PM" src="https://user-images.githubusercontent.com/12182961/109945987-c9d9db00-7cfd-11eb-819c-7d5974e37408.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 32 40 PM" src="https://user-images.githubusercontent.com/12182961/109947179-0bb75100-7cff-11eb-897f-7d7bfe4a44f4.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 32 46 PM" src="https://user-images.githubusercontent.com/12182961/109947183-0d811480-7cff-11eb-9bbc-10e3c6e9aaf2.png">
<img width="1680" alt="Screenshot 2021-03-04 at 3 36 00 PM" src="https://user-images.githubusercontent.com/12182961/109947516-6781da00-7cff-11eb-86b6-97920e2e6792.png">
<img width="1680" alt="Screenshot 2021-03-06 at 9 48 22 AM" src="https://user-images.githubusercontent.com/12182961/110194929-3b7e6a00-7e61-11eb-92f8-fad94449d43c.png">
<img width="1680" alt="Screenshot 2021-03-06 at 9 48 54 AM" src="https://user-images.githubusercontent.com/12182961/110194931-3de0c400-7e61-11eb-9ee2-d76d8915b0ec.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 14 06 AM" src="https://user-images.githubusercontent.com/12182961/110195681-fc521800-7e64-11eb-94c9-cec9b74743e0.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 14 36 AM" src="https://user-images.githubusercontent.com/12182961/110195682-fe1bdb80-7e64-11eb-875a-478183d5bfee.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 19 59 AM" src="https://user-images.githubusercontent.com/12182961/110195767-ae89df80-7e65-11eb-8c92-ab0ffef6a377.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 23 29 AM" src="https://user-images.githubusercontent.com/12182961/110195875-5c958980-7e66-11eb-86cc-bab612483df2.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 23 59 AM" src="https://user-images.githubusercontent.com/12182961/110195877-5e5f4d00-7e66-11eb-89e1-dbc43229bab2.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 24 22 AM" src="https://user-images.githubusercontent.com/12182961/110195879-5f907a00-7e66-11eb-81e6-c8071f70c5bd.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 24 29 AM" src="https://user-images.githubusercontent.com/12182961/110195883-628b6a80-7e66-11eb-94a3-8c14a0a8bf2f.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 25 36 AM" src="https://user-images.githubusercontent.com/12182961/110195885-64552e00-7e66-11eb-9315-365a37b139b8.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 28 39 AM" src="https://user-images.githubusercontent.com/12182961/110196008-8e5b2000-7e67-11eb-82c2-a5c2c88dce81.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 28 58 AM" src="https://user-images.githubusercontent.com/12182961/110196009-9024e380-7e67-11eb-8909-34dadd5903d4.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 30 36 AM" src="https://user-images.githubusercontent.com/12182961/110196011-91eea700-7e67-11eb-8e5e-092834f1999f.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 32 30 AM" src="https://user-images.githubusercontent.com/12182961/110196013-93b86a80-7e67-11eb-8ae6-d6ef270f111b.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 32 59 AM" src="https://user-images.githubusercontent.com/12182961/110196015-961ac480-7e67-11eb-8f2a-fbcfbfb022c8.png">
<img width="1680" alt="Screenshot 2021-03-06 at 10 33 52 AM" src="https://user-images.githubusercontent.com/12182961/110196016-97e48800-7e67-11eb-9b94-3bc217909964.png">

**Comment by HadesArchitect** [comment]
Very good job @jasbir84! Did you got grafana working? Can you connect using cqlsh?

**Comment by jasbir84** [comment]
@HadesArchitect Yeah Attached Screen shots of Grafana.

**Comment by jasbir84** [comment]
@HadesArchitect  Yes able to connect via cqlsh as well. attached screen shot for your reference. Now my local cluster is running fine without issues. Able to run the Spring Boot based application as well(Both Front End and Backend).

**Comment by jasbir84** [comment]
@HadesArchitect  All the Steps in the workshop has been completed now locally using Kind cluster :) Able to call stargate APIs as well. Also able to invoke repair on the cluster. Let me know in case any improvement is needed now.

**Comment by HadesArchitect** [comment]
Looks great! 😀

**Comment by HadesArchitect** [comment]
https://api.badgr.io/public/assertions/PTf6Q3wZTya41B2pR2FyUg Here we go! (Did you get an email?)
